pulsar-functions/utils/src/main/java/org/apache/pulsar/functions/utils/FunctionConfigUtils.java

```diff
@@ -373,6 +373,15 @@ public static FunctionConfig convertFromDetails(FunctionDetails functionDetails)
 
         functionConfig.setCleanupSubscription(functionDetails.getSource().getCleanupSubscription());
         functionConfig.setAutoAck(functionDetails.getAutoAck());
+
+        // Set subscription position
+        Function.SubscriptionPosition subPosition = functionDetails.getSource().getSubscriptionPosition();
+        if (subPosition == Function.SubscriptionPosition.EARLIEST) {
```

Comment:
Could you make the `Function.SubscriptionPosition.EARLIEST` before the equal symbol? Or use the equal method.

```
Function.SubscriptionPosition.EARLIEST == subPosition
```

or

```
Function.SubscriptionPosition.EARLIEST.equals(subPosition)
```

Reply:
thanks for the suggestion, applied with the latest commit.